packages/asset-swapper/src/utils/alt_mm_implementation_utils.ts

```diff
@@ -248,7 +248,7 @@ export async function returnQuoteFromAltMMAsync<ResponseT>(
         };
```

Comment:
since `asset-swapper` is sort of readonly now how about if we undo changes in `asset-swapper` and add it to `.prettierignore` instead?